src/terraform/dss-ci-cd/scripts/resume-runner.sh

```diff
@@ -0,0 +1,2 @@
+gcloud compute instances resume runner
```

Comment:
How about an optional argument to specify the project for which the runner should be resumed?

Because if you suspend runner A and then `engage-project` with project runner B, you'll need to resume A before you can `engage-project` with it